galcheat/utilities.py

```diff
@@ -55,7 +55,7 @@ def mag2counts(magnitude, survey, filter):
         filter = survey.get_filter(filter)
 
     flux = (magnitude - filter.zeropoint).to(u.electron / u.s)
-    counts = flux * filter.exposure_time
+    counts = flux * filter.full_exposure_time
```

Comment:
should we add to docs that full exposure time is used by default? and make the exposure time variable? Or is this a separate PR?

Reply:
Separate PR, I have already an issue about this but wanted to merge this first.